tests/containers/validate_btrfs.pm

```diff
@@ -88,13 +95,32 @@ sub _test_btrfs_device_mgmt {
 }
 
 sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+    die "Module requires two disks to run" unless check_var('NUMDISKS', 2);
+    my ($running_version, $sp, $host_distri) = get_os_release;
+    install_docker_when_needed($host_distri);
+    allow_selected_insecure_registries(runtime => 'docker');
     my $docker    = containers::runtime->new(runtime => 'docker');
     my $btrfs_dev = '/var/lib/docker';
-    _sanity_test_btrfs($docker, $btrfs_dev);
+    my ($image_names, $stable_names) = get_suse_container_urls();
+    _sanity_test_btrfs($docker, $btrfs_dev, $stable_names->[0]);
     _test_btrfs_thin_partitioning($docker, $btrfs_dev);
     _test_btrfs_device_mgmt($docker, $btrfs_dev);
     _test_btrfs_balancing($btrfs_dev);
     $docker->cleanup_system_host;
 }
 
+sub post_fail_hook {
```

Comment:
we really need a base class for containers :)

Reply:
i would say that it is a good thing to have and this doesnt change the things as it is so far.

Reply:
something like this https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12199 ? :)

Reply:
@asmorodskyi why file from #12199?